build.gradle

```diff
@@ -7,18 +7,18 @@ plugins {
 
 repositories {
 	mavenLocal()
-	maven {
-		name = "CurseForge"
-		url = "https://minecraft.curseforge.com/api/maven"
-	}
-	maven {
-		name = "GitHub"
-		url = "https://maven.pkg.github.com"
-	}
-	maven {
-		name = "JitPack"
-		url = "https://jitpack.io"
-	}
+//	maven {
+//		name = "CurseForge"
+//		url = "https://minecraft.curseforge.com/api/maven"
+//	}
+//	maven {
+//		name = "GitHub"
+//		url = "https://maven.pkg.github.com"
+//	}
+//	maven {
+//		name = "JitPack"
+//		url = "https://jitpack.io"
+//	}
```

Comment:
These mavens make my configuartion slow as fuck (costs me extra 30 seconds configuring gradle) and breaks the source jar remapping, so I am removing them